MMM-Bring.js

```diff
@@ -135,6 +153,18 @@ Module.register("MMM-Bring", {
             this.sendSocketNotification("GET_LIST", this.config);
         }
     },
+    
+    notificationReceived: function(notification, payload) {
+        if (notification === "KEYBOARD_INPUT" && payload.key === "bring" && payload.message != '') {
```

Comment:
payload.key could be "more unique" with mmm-bring but this one is fine as well

Reply:
Changed to "mmm-bring"